src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java

```diff
@@ -586,7 +586,7 @@ public void verifyTOCsectionLinkWorks(int numberOfTheSection) {
   }
 
   public void verifyWikiTitleOnCongratualtionsLightBox(String wikiName) {
-    wait.forElementVisible(welcomeLightBoxTitle);
+    wait.forElementVisible(welcomeLightBoxTitle, 30);
```

Comment:
Can we move `30` to the `const` with some explicit name? Or at least leave the comment here why `30`